src/app/components/Labels/LabelSortableList.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { c } from 'ttag';
+import { SortableContainer } from 'react-sortable-hoc';
+import { Icon } from 'react-components';
+
+import LabelSortableItem from './LabelSortableItem';
+
+export default SortableContainer(({ items, onEditLabel, onRemoveLabel, onToggleChange }) => {
+    return (
+        <table className="pm-simple-table noborder border-collapse mt1">
```

Comment:
Use basic component instead of native HTML. If few props are missing, feel free to add it.